tests/unit/disco/containers/TestDiscoPane.js

```diff
@@ -185,6 +186,7 @@ describe(__filename, () => {
           windows: undefined,
         },
         isRestartRequired: false,
+        themeData,
```

Comment:
ugh, this is an old test and it's not very useful. It may be worth just using `createInternalAddon({ ...fakeDiscoAddon, ... })` for the assertions here.

Reply:
Looks like we are not using the internal addon representation here, or I don't understand how it works right now :/

Reply:
ah, should be ok now.